lrmodule/helpers.py

```diff
@@ -0,0 +1,54 @@
+import numpy as np
+
+
+def select_marktype_ccf(p: np.ndarray) -> np.ndarray:
+    """Select the 'ccf' marktype from the input features.
+
+    As there is only one feature, this function just returns the input as is.
+    """
+    return p
+
+
+def transform_marktype_ccf(original):
+    """Transform the 'ccf' marktype using a log transformation.
+
+    The input values are expected to be in the range [-1, 1]. They are first scaled to [0, 1], and then the logit'ed..
```

Comment:
```suggestion
    The input values are expected to be in the range [-1, 1]. They are first scaled to [0, 1], after which a logit transformation is applied.
```

"logit'ed" is misschien een beetje vaag, wat vind je van deze suggestie? Ik weet niet of "transformation" het juiste woord is.